app/src/common/shared/com/igalia/wolvic/browser/LoginStorage.kt

```diff
@@ -22,7 +23,11 @@ class LoginStorage(
     init {
         EngineProvider.getOrCreateRuntime(context).setUpLoginPersistence(places.logins)
         GlobalScope.launch(Dispatchers.IO) {
-            places.logins.value.warmUp()
+            try {
+                storage.value.warmUp()
+            } catch (e: InvalidKeyException) {
+                storage.value.wipeLocal()
```

Comment:
I think this deserves a brief comment explaining why we have to do this. We could include links to already existing reports for mozilla components like https://github.com/mozilla-mobile/android-components/issues/6681or https://github.com/mozilla-mobile/fenix/issues/15597.